addons/website_blog/views/snippets/s_latest_posts.xml

```diff
@@ -1,33 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-<template id="s_latest_posts" name="Latest Posts - List">
-    <section class="s_latest_posts pt16 pb16">
+<!-- Snippet -->
+<template id="s_latest_posts" name="Blog Posts">
+    <section class="s_latest_posts pt16 pb16" data-vcss="001">
         <div class="container">
-            <h2>Latest Posts</h2>
-            <div class="row s_col_no_bgcolor s_nb_column_fixed js_get_posts"
+            <div class="row s_col_no_bgcolor s_nb_column_fixed js_get_posts o_latest_posts_big_picture effect-marley"
```

Comment:
Let's review the class name to follow our conventions: s_latest_posts_XXX even for effect stuff.